static/app/gettingStartedDocs/dotnet/maui.tsx

```diff
@@ -134,82 +139,96 @@ const onboarding: OnboardingConfig = {
   verify: params => [
     {
       type: StepType.VERIFY,
-      description: t(
-        'To verify your set up, you can capture a message with the SDK, anywhere in your code after the application is built, such as in a page constructor or button click event handler:'
-      ),
-      configurations: [
+      content: [
         {
+          type: 'text',
+          text: t(
+            'To verify your set up, you can capture a message with the SDK, anywhere in your code after the application is built, such as in a page constructor or button click event handler:'
+          ),
+        },
+        {
+          type: 'code',
           language: 'csharp',
           code: 'SentrySdk.CaptureMessage("Hello Sentry");',
         },
       ],
     },
     ...(params.isPerformanceSelected
-      ? [
+      ? ([
           {
             title: t('Tracing'),
-            description: (
-              <Fragment>
-                {t(
+            content: [
+              {
+                type: 'text',
+                text: t(
                   'We do not yet have automatic performance instrumentation for .NET MAUI. We will be adding that in a future release. However, if desired you can still manually instrument parts of your application.'
-                )}
-                <p>
-                  {tct(
-                    'For some parts of your code, [automaticInstrumentationLink:automatic instrumentation] is available across all of our .NET SDKs, and can be used with MAUI as well:',
-                    {
-                      automaticInstrumentationLink: (
-                        <ExternalLink href="https://docs.sentry.io/platforms/dotnet/guides/maui/tracing/instrumentation/automatic-instrumentation/" />
-                      ),
-                    }
-                  )}
-                </p>
-              </Fragment>
-            ),
-            configurations: [
+                ),
+              },
+              {
+                type: 'text',
+                text: tct(
+                  'For some parts of your code, [automaticInstrumentationLink:automatic instrumentation] is available across all of our .NET SDKs, and can be used with MAUI as well:',
```

Comment:
Your conversion is correct. I’m just wondering if we should use a period instead of a colon here, since having two colons in consecutive phrases looks a bit odd.

<img width="907" height="65" alt="Image" src="https://github.com/user-attachments/assets/94003221-e0dd-49f4-a680-f88701db0e92" />